Cargo.toml

```diff
@@ -12,10 +12,12 @@ categories = ["network-programming"]
 edition = "2018"
 
 [dependencies]
+mio = "0.6"
```

Comment:
Tokio requires mio, why do we have to list it separately?

Reply:
I explicitly needed it in some earlier code, but its now safe to remove. Good catch!

Reply:
Removed